src/main/java/commands/AddCommand.java

```diff
@@ -0,0 +1,66 @@
+package commands;
+
+import functions.*;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import tasks.*;
```

Comment:
As commented in other parts of the code, you might consider importing specific classes instead of using the "*" notation?